dallinger/frontend/static/scripts/dallinger.js

```diff
@@ -266,14 +264,17 @@ var submitNextResponse = function (n) {
         },
         success: function() {
             if (n <= ids.length) {
-                submitNextResponse(n + 1);
+                submitNextResponse(n + 1, callback);
+            } else {
+                callback();
             }
         },
         error: function (err) {
             var errorResponse = JSON.parse(err.response);
             if (errorResponse.hasOwnProperty("html")) {
                 $("body").html(errorResponse.html);
```

Comment:
It seems kind of extreme to replace the entire page with the html from the error response. Is there a particular use case for doing this? It also seems strange that this stops submitting subsequent responses (though it has potentially successfully submitted prior ones), but still calls the `callback` function.

Reply:
This question is for @suchow or @vlall I think.